src/main/java/mil/dds/anet/database/LocationDao.java

```diff
@@ -53,6 +54,33 @@ public int updateInternal(Location l) {
         .bind("status", DaoUtils.getEnumId(l.getStatus())).execute();
   }
 
+  @InTransaction
+  public int mergeLocation(Location loserLocation, Location mergedLocation) {
+    // Update old locations relations with merged location.
+    getDbHandle()
+        .createUpdate("/* updateNotesOfLoserLocationAfterMerge */ UPDATE \"noteRelatedObjects\" "
+            + "SET \"relatedObjectUuid\" = :relatedObjectUuid WHERE \"relatedObjectUuid\" = :oldRelatedObjectUuid")
+        .bind("oldRelatedObjectUuid", loserLocation.getUuid())
+        .bind("relatedObjectUuid", mergedLocation.getUuid()).execute();
```

Comment:
This will fail if there's a note pointing to *both* locations. See https://github.com/NCI-Agency/anet/blob/GH-2994-improve-and-extend-merge-for-locations-object/src/main/java/mil/dds/anet/database/PersonDao.java#L407-L418 in `PersonDao::mergePeople` for a solution.